alpine-php/k8s/etc/msmtprc

```diff
@@ -0,0 +1,13 @@
+# Base configuration for for msmtp.
```

Comment:
Do you still use `msmtp`? No decent smtp drupal module? :(

Reply:
Definitely don't want to use Drupal smtp modules.

Reply:
Sad.

Reply:
No, happy! This way sendgrid config is hidden from anyone not ops and easily managed.